### DISCUSSION TITLE
Where does the langflow stores its artifacts ?

### DISCUSSION THREAD

**Discussion by n-chilka** [langflow-ai/langflow#912] [Q&A]
Artifacts I mean here is if I delete everything, by storing artifacts only and once I install again and will paste my artifacts then all the previous state should be return.

I want to know the artifacts path.

**Comment by ogabrielluiz** [comment]
Hey

I'm not sure I follow what you mean by artifacts.

Langflow uses the user cache dir(e.g. ~/.cache/langflow)  to store the database (by default) and other cached information.
The flow itself is saved in the database, but you can download your collection, resulting in a JSON file.
